Assets/MRTK/SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -1605,6 +1624,12 @@ public void OnInputDown(InputEventData eventData)
 
             if (ShouldListenToUpDownEvent(eventData))
             {
+                if (eventData.used)
```

Comment:
It could be useful to explain here, and elsewhere, why we reset the input tracking states when the event data is used.